WireUI/Sources/WireMoveToFolderUI/Protocols/UpdateConversationFolderUseCaseType.swift

```diff
@@ -16,11 +16,14 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
+import Foundation
+
 /// Protocol for handling moving a conversation to a specified folder
-public protocol MoveConversationToFolderUseCaseType: Sendable {
+public protocol UpdateConversationFolderUseCaseType {
```

Comment:
Why do we use the `Type` suffix and not `Protocol` like we usually do?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/46b118cf3de10921f17ce8a1307610ebd3452cd1